addons/product/models/product_pricelist.py

```diff
@@ -37,6 +37,11 @@ def _get_default_item_ids(self):
     country_group_ids = fields.Many2many('res.country.group', 'res_country_group_pricelist_rel',
                                          'pricelist_id', 'res_country_group_id', string='Country Groups')
 
+    discount_policy = fields.Selection([
```

Comment:
No change of model in a frozen version!
cc @KangOl

Reply:
Please, add this feature before release V12